suse_caasp

```diff
@@ -226,10 +226,10 @@ if check_rpm kubernetes-client && [ -f /etc/kubernetes/admin.conf ]; then
     log_cmd $OF 'kubectl version'
     log_cmd $OF 'kubectl api-versions'
     log_cmd $OF 'kubectl config view'
-    log_cmd $OF 'kubectl -n kube-system get pods'
+    log_cmd $OF 'kubectl get pods -n kube-system -o wide'
     log_cmd $OF 'kubectl get events --sort-by=.metadata.creationTimestamp'
-    log_cmd $OF 'kubectl get nodes'
-    log_cmd $OF 'kubectl get all -A'
+    log_cmd $OF 'kubectl get nodes -o wide'
+    log_cmd $OF 'kubectl get all -A -o wide'
     log_cmd $OF 'kubectl get nodes -o yaml'
```

Comment:
`kubectl get nodes -o wide` and `kubectl get nodes -o yaml` are both used. Do you need both ?

`kubectl get nodes -o yaml` already gives more info than `kubectl get nodes -o wide`.

`kubectl get nodes -o wide` does not add more info but it's a good overview.

Reply:
nervermind, they both have use here